packages/ds-healthcare-gov/src/components/Header/ActionMenu.test.tsx

```diff
@@ -89,15 +90,15 @@ describe('ActionMenu', function () {
         ],
       });
       fireEvent.click(screen.getByText('ZOMBO'));
-      expect(window.utag.link.mock.calls[0][0]).toMatchSnapshot();
+      expect(mock).toHaveBeenCalled();
     });
 
     it('sends analytics event when logged-out menu opened', () => {
       renderForAnalytics({
         links: [{ label: 'label', href: 'href' }],
       });
       fireEvent.click(screen.getByRole('button'));
-      expect(window.utag.link.mock.calls[0][0]).toMatchSnapshot();
+      expect(mock).toHaveBeenCalled();
```

Comment:
Why were these snapshots removed?

Reply:
oh good point I forgot about these, when I was working on them I had trouble finding these items within the rtl way of doing things.

Reply:
Ok @pwolfert these have been re-added

Reply:
It looks like the snapshots that were added back in have all calls for all test instances, so they're really big and gnarly. The mock on `window.utag.link` doesn't reset between tests.

Reply:
Very true, took me a minute to figure this out, managed to grab only the needed data, updated!

Reply:
Whoops but now it's grabbing the wrong one.. looking into it.

Reply:
Ok actually fixed these, sorry for the noise.